### ISSUE TITLE
Type Errors on Python 3.4.3

### ISSUE THREAD

**Issue by pampersrocker** [msiemens/PyGitUp#25]
When installing PyGitUp on Python 3.4.3 i get the following error when showing the version running `git up --version`:
```
 Marvin@M:/git/g3-engine (cub3d-version) ?1
[0:18:48] » git up --version
GitUp version is: v1.3.0
Checking for updates...Traceback (most recent call last):
  File "C:\Python34\Scripts\git-up-script.py", line 9, in <module>
    load_entry_point('git-up==1.3.0', 'console_scripts', 'git-up')()
  File "C:\Python34\Scripts\gitup.py", line 580, in run
    GitUp(sparse=True).version_info()
  File "C:\Python34\Scripts\gitup.py", line 394, in version_info
    details = json.load(urlopen(PYPI_URL))
  File "C:\Python34\lib\json\__init__.py", line 268, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "C:\Python34\lib\json\__init__.py", line 312, in loads
    s.__class__.__name__))
TypeError: the JSON object must be str, not 'bytes'
←[0m%
```

Also a similar error occurs while fast-forwarding a branch:
```
 Marvin@M:/git/g3-engine (cub3d-version)
[0:19:24] » git up
Fetching epic
Fetching origin
cub3d-version                fast-forwarding...
Traceback (most recent call last):
  File "C:\Python34\Scripts\git-up-script.py", line 9, in <module>
    load_entry_point('git-up==1.3.0', 'console_scripts', 'git-up')()
  File "C:\Python34\Scripts\gitup.py", line 590, in run
    gitup.run()
  File "C:\Python34\Scripts\gitup.py", line 203, in run
    self.rebase_all_branches()
  File "C:\Python34\Scripts\gitup.py", line 288, in rebase_all_branches
    self.log(branch, target)
  File "C:\Python34\Scripts\gitup.py", line 355, in log
    bat_file.file.write('@echo off\n')
TypeError: 'str' does not support the buffer interface
←[0m%
```

No problems on python 2.7.9

**Comment by msiemens** [comment]
Thanks for reporting this @pampersrocker. This should be fixed in 71dd44d. Can you confirm that?

**Comment by pampersrocker** [comment]
Sorry for the late respone, but yes its fixed now. Thanks!
